### ISSUE TITLE
SItes on your VPS won't load

### ISSUE THREAD

**Issue by ljcool2006** [fukionline/fukionline#1]
oddly enough this doesn't affect the ftp

**Comment by fukionline** [comment]
i noticed. i'll try a restart

**Comment by fukionline** [comment]
assuming it's an issue with apache
let me know if it works

**Comment by fukionline** [comment]
>downright awesome screen
hmmm... if that stays up i might need to do more drastic measures, since i can't access the vps

**Comment by fukionline** [comment]
Yeah, looks like it's still not loaded. I could straight reinstall the VPS, but that would be a lot of work, to set everything back up.

**Comment by ljcool2006** [comment]
> > downright awesome screen
> 
> hmmm... if that stays up i might need to do more drastic measures, since i can't access the vps

do you still have the hard drive that has the passwords? try to find them in there

**Comment by fukionline** [comment]
> > > downright awesome screen
> > 
> > 
> > hmmm... if that stays up i might need to do more drastic measures, since i can't access the vps
> 
> do you still have the hard drive that has the passwords? try to find them in there

i don't, thats why i cant get in

**Comment by ljcool2006** [comment]
do you still have the databases? i can't connect to them rn because the web interface is down and i can't connect via another client either (presumably because my machine isn't whitelisted)

**Comment by fukionline** [comment]
nope :/
i'll try contacting support

**Comment by ljcool2006** [comment]
ok, be sure to update me

**Comment by ljcool2006** [comment]
bump

**Comment by ljcool2006** [comment]
it's been almost a month

**Comment by ljcool2006** [comment]
@fukionline
